packages/block-editor/src/components/block-tools/block-toolbar-popover.js

```diff
@@ -101,7 +101,7 @@ export default function SelectedBlockTools( {
 				{ ...popoverProps }
 			>
 				{ shouldShowContextualToolbar && (
-					<BlockContextualToolbar
+					<BlockToolbar
```

Comment:
I don't understand why the change of component here. Or are you saying `BlockToolbarPopover` is the component responsible for adding the "contextual" specific styles...

Reply:
> Or are you saying BlockToolbarPopover is the component responsible for adding the "contextual" specific styles...

Exactly. I think renaming it to Popover over Contextual better communicates the purpose of the component since "contextual" is a meaningful word for the top toolbar as well. Popover is more specific to the floating toolbar.